source/jormungandr/jormungandr/interfaces/v1/fields.py

```diff
@@ -881,6 +881,11 @@ def output(self, key, obj):
 instance_status_with_parameters['traveler_profiles'] = fields.List(fields.Nested(instance_traveler_types,
                                                                                  allow_null=True))
 
+impacted_line_section = {
+    'from': NonNullNested(stop_area),
```

Comment:
And I think a pt_object will be better, if we want to use stop_point in the future